### DISCUSSION TITLE
Neper -V -datacellcolscheme ipf, angle, rodrigues etc.

### DISCUSSION THREAD

**Discussion by xshang93** [neperfepx/neper#367] [Q&A] [answered]
Hi again @rquey ! 

When generating images from neper, it is allowed to choose 'ipf(dir)' as the color scheme. However, I notice that cubic symmetry is assumed when using the scheme for crystal orientation. Is there a way to visualize in one figure for both cubic and hexagonal symmetry to make a plot mimicking this one below?

![10 37188-lam 2021 016-Figure6 (1)](https://user-images.githubusercontent.com/86078353/150293526-b6728df6-5b5b-421f-9735-0c0d7529525a.jpg)

**Comment by rquey** [comment, marked as answer]
No, this is not possible at the moment... (you can make a [feature request](https://github.com/rquey/neper/issues/new?assignees=&labels=&template=feature_request.md&title=)).
